static/reference/next/client/README.md

```diff
@@ -1 +1,28 @@
-> Docs for Pulsar Java Client Configuration.
+## Client Configurations
+
+If you create a Java client, producer, consumer, or reader, you can use the `loadConf` configurations.
+
+- [Client configurations](/next/client/client-configuration-client)
```

Comment:
@urfreespace  I'm wondering whether we can use `@pulsar:version_origin@` to replace `/next`? This cannot be tested out in my local build.

Reply:
Using the new var `@pulsar:version_reference@` to replace `next`.